src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp

```diff
@@ -455,7 +455,9 @@ ov::runtime::Tensor generate(const std::shared_ptr<ngraph::op::v1::GatherTree>&
             return ov::test::utils::create_and_fill_tensor(elemType, targetShape, inGenData.range, inGenData.start_from, inGenData.resolution, inGenData.seed);
         }
         default:
-            return generate(std::dynamic_pointer_cast<ov::Node>(node), port, elemType, targetShape);
+            InputGenerateData inGenData;
```

Comment:
Why the change was required?

Reply:
test requires changes in input data generation

Reply:
generate(...) creates tensor with default values. I need `range` equal to `maxBeamIndx`

Reply:
Could you please explain the details? Anyway, we can change the `generate` signature as was discussed

Reply:
I aligned behavior with legacy tests. Otherwise tests fail